web/console/src/app/views/WhitePaperPage/index.scss

```diff
@@ -6,10 +6,61 @@
     color: white;
 
     &__wrapper {
-        width: 75%;
-        min-height: 100vh;
-        margin-left: 25%;
-        background-color: $mainBackground;
-        padding: 100px;
+        width: 70%;
+        min-height: calc(100vh - 98px);
+        margin-left: 30%;
+        background-color: #8db9ff;
+        padding: 100px 100px 0;
+    }
+
+    .summary,
+    .game-mechanics,
+    .pay-to-earn-economy,
+    .technology {
+
+        &__title {
+            margin-bottom: 43px;
+            font-family: "Orbitron", sans-serif;
+            font-style: normal;
+            font-weight: 900;
+            font-size: 34px;
+            line-height: 43px;
+            letter-spacing: 0.12em;
+            text-transform: uppercase;
+            color: #fff;
+        }
+
+        &__description {
+            margin-bottom: 43px;
+            font-size: 17px;
+            line-height: 30px;
+            font-family: "Orbitron", sans-serif;
+            font-style: normal;
+            font-weight: 900;
+            letter-spacing: 0.12em;
+            text-transform: uppercase;
+            color: #fff;
+        }
+    }
```

Comment:
`
   .summary,
    .game-mechanics,
    .pay-to-earn-economy,
    .technology {
            font-family: "Orbitron", sans-serif;
            font-style: normal;
            font-weight: 900;

        &__title {
            margin-bottom: 43px;
            font-size: 34px;
            line-height: 43px;
            letter-spacing: 0.12em;
            text-transform: uppercase;
            color: #fff;
        }

        &__description {
            margin-bottom: 43px;
            font-size: 17px;
            line-height: 30px;
            letter-spacing: 0.12em;
            text-transform: uppercase;
            color: #fff;
        }
    }`